cli/version.go

```diff
@@ -0,0 +1,79 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/coder/coder/buildinfo"
+	"github.com/coder/coder/cli/clibase"
+	"github.com/coder/coder/cli/cliui"
+)
+
+// version prints the coder version
+func (*RootCmd) version() *clibase.Cmd {
+	handleHuman := func(inv *clibase.Invocation) error {
+		var str strings.Builder
```

Comment:
nit: do you need a string builder if there is no way to break execution in the middle of processing?